modules/rostests/winetests/kernel32/generated.c

```diff
@@ -5,9 +5,11 @@
  * Unit tests for data structure packing
  */
 
+#ifndef __REACTOS__
```

Comment:
Can this be fixed in CMake instead?
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=remove_definitions

Reply:
remove_definitions of course won't work here since it's already being used

Replacing
```
add_definitions(-DWINVER=0x600 -D_WIN32_WINNT=0x601)
```
with
```
set_source_files_properties(${SOURCE} PROPERTIES COMPILE_DEFINITIONS "WINVER=0x600;_WIN32_WINNT=0x601")
```
and removing generated.c from the SOURCE list might work.

Reply:
I don't understand what would need to be fixed.
`#ifndef` is needed as this is WineSync and as the other generated.c (except 1) have.
This PR goal is to re-enable the test only, all further work should/shall happen separately/later. (I am working on it on both Wine and ReactOS.)

Reply:
You're making changes to the source code (a "ROS-Diff") where a CMake change can solve the same problem. Leaving 3rd party files untouched is generally preferable to introducing `__REACTOS__` ifdefs.

Reply:
That seems like a good rule, which I mostly agree with. Although it can be misleading when reading the test file itself.
Nonetheless, this PR is not the place to investigate/start applying it (to all 10 files): feel free to fix that yourself separately.